src/membrane_proof.rs

```diff
@@ -163,12 +163,8 @@ pub fn delete_mem_proof_file() -> Result<()> {
 /// If a membrane proof is already generated downloads that membrane proof.
 /// from HBS server and returns as a string
 async fn download_memproof(admin: Admin) -> Result<String> {
-    let mut role = "host".to_string();
-    if let Ok(bool) = env::var("IS_HOLOFUEL_INSTANCE") {
-        if bool == "true" {
-            role = "holofuel".to_string()
```

Comment:
Isn't holofuel role already in use by holofuel users? Perhaps this should be called something like `holo` or `infrastructure`?